tests/pop_cmaes_test.cpp

```diff
@@ -0,0 +1,133 @@
+#include <ensmallen.hpp>
```

Comment:
```suggestion
/**
 * @file pop_cmaes_test.cpp
 * @author Benjami Parellada
 *
 * Tests for the POP_CMAES class, including IPOP CMA-ES and BIPOP CMA-ES.
 *
 * ensmallen is free software; you may redistribute it and/or modify it under
 * the terms of the 3-clause BSD license.  You should have received a copy of
 * the 3-clause BSD license along with ensmallen.  If not, see
 * http://www.opensource.org/licenses/BSD-3-Clause for more information.
 */
#include <ensmallen.hpp>
```
We should include a license header here too.  (I put your name here just as a guess, but you could remove it if for some reason you want to :))